tests/src/test/java/alluxio/master/file/ConcurrentDeleteIntegrationTest.java

```diff
@@ -78,19 +75,10 @@
           "sleep://" + mLocalUfsPath).setProperty(PropertyKey
           .USER_FILE_MASTER_CLIENT_THREADS, CONCURRENCY_FACTOR).build();
 
-  // Must be done in beforeClass so execution is before rules
-  @BeforeClass
-  public static void beforeClass() throws Exception {
-    SleepingUnderFileSystemOptions options = new SleepingUnderFileSystemOptions();
-    sSleepingUfsFactory = new SleepingUnderFileSystemFactory(options);
-    options.setDeleteFileMs(SLEEP_MS).setDeleteDirectoryMs(SLEEP_MS);
-    UnderFileSystemFactoryRegistry.register(sSleepingUfsFactory);
-  }
-
-  @AfterClass
-  public static void afterClass() throws Exception {
-    UnderFileSystemFactoryRegistry.unregister(sSleepingUfsFactory);
-  }
+  @ClassRule
+  public UnderFileSystemFactoryRegistryRule mUnderFileSystemFactoryRegistry =
```

Comment:
Note that `beforeClass()` and `afterClass()` are also static, 
check more details https://stackoverflow.com/questions/1052577/why-must-junits-fixturesetup-be-static